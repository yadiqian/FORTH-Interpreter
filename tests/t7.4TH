3 2.5 5 * - . CR 45.0 15 / 270 3 / EMIT CR . world hello CONCAT2 