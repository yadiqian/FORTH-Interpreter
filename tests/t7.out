-9.5
Z
3.0
Stack is not empty
[Id "helloworld"]